modules/countdown.js

```diff
@@ -3,46 +3,78 @@ require('moment-countdown')
 
 let timeout = null
 
-const ARTICLE_50 = '2020-01-31T00:00:00+01:00'
+function isPast (date) {
+  const now = moment()
+  const expiryDate = moment(date)
+  return expiryDate.isBefore(now)
+}
+
+function nextDate (milestones) {
+  for (const milestone in milestones) {
+    if (!isPast(milestone.date)) return milestone
+  }
+  return { name: null, date: null }
```

Comment:
I'm not sure you need this; just return `null` or `undefined`, and when you actually go to use it check for its presence first -- or simply `throw new Error('No date available')`